examples/js/loaders/MTLLoader.js

```diff
@@ -449,6 +459,46 @@ THREE.MTLLoader.MaterialCreator.prototype = {
 
 		this.materials[ materialName ] = new THREE.MeshPhongMaterial( params );
 		return this.materials[ materialName ];
+	},
+
+	getTextureParams: function( value, matParams ) {
+
+		texParams = {
+
+			scale: new THREE.Vector2( 1, 1 ),
+			offset: new THREE.Vector2( 0, 0 ),
+
+		 };
+
+		var items = value.split(/\s+/);
+		var pos;
+
+		pos = items.indexOf('-bm');
+		if (pos >= 0) {
+
+			matParams.bumpScale = parseFloat( items[1] );
```

Comment:
> Also matParams is not provided by the call sites, nor inited to empty object if not defined. Will be undefined here and throw and exception.
> 
> You probably wanted to init textParams.bumpScale to 1,1 and set it here.

Yes, this is actually how I implemented it before .. but I'd say it is better to pass `params` as a parameter, as then we can avoid a) setting a default (which is defined in `Material` anyway) and b) copying it back to `params`. (The only reason we need the `texParams` temporary is because we don't yet have a `Texture` instance at this point)